backend/pkg/keywords/upload.go

```diff
@@ -37,15 +50,22 @@ func (s *Service) Upload(c *gin.Context) {
 	reader := csv.NewReader(fileTmp)
 
 	rawRecords, _ := reader.ReadAll()
+
 	keyword := models.ExtractedKeywords{}
 
 	keyword.Dehydrate(rawRecords)
 
+	if len(keyword.Data) > 100 {
```

Comment:
For code discovery (understanding business logic) and maintainability concerns, it would be best to define the value `100` as a constant or config variables instead of being hard-coded.